osquery/tables/events/darwin/openbsm_events.cpp

```diff
@@ -56,6 +59,13 @@ class OpenBSMExecVESubscriber : public EventSubscriber<OpenBSMEventPublisher> {
 
   Status Callback(const OpenBSMEventContextRef& ec,
                   const OpenBSMSubscriptionContextRef& sc);
+
+ private:
+  Status handleExec(const OpenBSMEventContextRef& ec);
+  Status handleFork(const OpenBSMEventContextRef& ec);
+  Status handleExit(const OpenBSMEventContextRef& ec);
+
+  std::unordered_map<uint32_t, uint32_t> ppid_map;
```

Comment:
Can/should we reserve a size?

Reply:
I'm not sure, I was investigating and the `kern.maxproc` can be at most 2500 for recent versions of macos, so at most we'll have 2500 entries which doesn't sound like a lot. What do you think?